### DISCUSSION TITLE
Does the __instance field allows me to access private variables?

### DISCUSSION THREAD

**Discussion by G-M-twostay** [pardeike/Harmony#539] [Q&A]
According to https://harmony.pardeike.net/articles/patching-injections.html#common-injected-values if I'm patching a instance method I can get the instance of it by changing the first argument to __instance. In my scenario I need the instance to access and modify some variables in the original class, which includes some private ones. Do I need to use reflect to manually get those variables or is this __instance already made them accessible?

**Comment by pardeike** [comment]
Sorry but GitHub issues are used for bugs and improvement suggestions. For usage discussions please use the official discord server.
